ogr/services/gitlab/pull_request.py

```diff
@@ -118,16 +119,50 @@ def create(
         source_branch: str,
         fork_username: str = None,
     ) -> "PullRequest":
-        mr = project.gitlab_repo.mergerequests.create(
-            {
-                "source_branch": source_branch,
-                "target_branch": target_branch,
-                "title": title,
-                "description": body,
-            }
-        )
+        repo = project.gitlab_repo
+        parameters = {
+            "source_branch": source_branch,
+            "target_branch": target_branch,
+            "title": title,
+            "description": body,
+        }
+
+        if project.is_fork and (
+            fork_username is None or fork_username == project.namespace
+        ):
+            parameters["target_project_id"] = project.parent.gitlab_repo.attributes[
+                "id"
+            ]
+        elif fork_username and fork_username != project.namespace:
```

Comment:
Maybe some comment about `else` variant?

Reply:
I'll add those, also usage as in github implementation